runtime_test.go

```diff
@@ -3014,6 +3014,33 @@ func TestExceptionStringifyError(t *testing.T) {
 			},
 			expectedString: "undefined",
 		},
+		{
+			description: "Array",
+			exceptionVal: func() Value {
+				array := runtime.builtin_newArray([]Value{
```

Comment:
Calvin, I tagged you just because I saw your name around Arrays and objects when I was trying to figure out how to construct stuff. Feel free to remove yourself if you don't think this needs a look